app/views/latest/index.html.erb

```diff
@@ -1,71 +1,79 @@
-<% page_title t("document.latest_from", subject: subject) %>
+<% page_title t("document.latest") %>
```

Comment:
Seeing this hidden menu on integration

![image](https://user-images.githubusercontent.com/71266765/135067141-584b453a-9b98-49fc-bede-92f13c037429.png)